library/keystone_user.py

```diff
@@ -50,7 +50,7 @@
      description:
         - The keystone url for authentication
      required: false
-     default: 'http://127.0.0.1:35358/v2.0/'
+     default: 'http://127.0.0.1:35357/v2.0/'
```

Comment:
so I'm not quite sure why this is changing. When the endpoints changed, the tasks that used these modules grew arguments to define the endpoint when using this module. Do you happen to have notes anywhere about what call to this module was failing?